target/linux/mvebu/image/cortexa72.mk

```diff
@@ -52,6 +52,20 @@ define Device/marvell_macchiatobin-singleshot
 endef
 TARGET_DEVICES += marvell_macchiatobin-singleshot
 
+define Device/marvell_clearfog-gt-8k
+  $(call Device/Default-arm64)
+  DEVICE_VENDOR := SolidRun
+  DEVICE_MODEL := Clearfog
+  DEVICE_VARIANT := GT-8K
+  DEVICE_ALT0_VENDOR := SolidRun
+  DEVICE_ALT0_MODEL := Clearfog
+  DEVICE_ALT0_VARIANT := GT-8K
```

Comment:
Wait, don't all of those match the `DEVICE_{VENDOR,MODEL,VARIANT}` from above?

Reply:
They do. I admit to a loss in creativity, should they just be marvel?

Reply:
I will drop the DEVICE_ALT0_*